docs/ovms_quickstart.md

```diff
@@ -37,21 +37,24 @@ docker pull openvino/model_server:latest
 
 ### Step 3: Provide a Model
 
-Store components of the model in the `model/1` directory. Here is an example command using curl and a face detection model:
+Store components of the model in the `models/1` directory. Here is an example command using curl and a face detection model:
```

Comment:
Why changing model/1 to models/1? As far I see it is incorrect, below: `mkdir -p model/1`